dffml/util/cli.py

```diff
@@ -222,10 +231,12 @@ class ModelCMD(CMD):
     Set a models model dir.
     '''
 
-    arg_model = Arg('-model', help='Model used for ML',
-            action=ParseModelAction, required=True)
     arg_model_dir = Arg('-model_dir', help='Model directory for ML',
             default=os.path.join(os.path.expanduser('~'), '.cache', 'dffml'))
+    arg_model = Arg('-model', help='Model used for ML',
```

Comment:
nit: Lets leave this line where it was. Since there's no reason to move it. The reason for me saying this is that when someone wants to see what each commit did they might do `git log -p` and the changes will be more clear to them if they don't see things change that don't make sense why they would have changes within the context of that commit.

Reply:
Sorry for this. I've made the changes :+1: